operators/multiclusterobservability/api/v1beta2/multiclusterobservability_types.go

```diff
@@ -48,6 +51,61 @@ type MultiClusterObservabilitySpec struct {
 // +kubebuilder:validation:Enum:={"default","minimal","small","medium","large","xlarge","2xlarge","4xlarge"}
 type TShirtSize string
 
+type LogsCollectionSpec struct {
+	// +optional
+	Enabled bool `json:"enabled,omitempty"`
+}
+
+type PlatformCapabilitiesSpec struct {
+	// +optional
+	Logs LogsCollectionSpec `json:"logs,omitempty"`
+}
+
+type ClusterLogForwarderSpec struct {
+	// +optional
+	Enabled bool `json:"enabled,omitempty"`
+}
+
+type UserWorkloadLogsSpec struct {
+	ClusterLogForwarder ClusterLogForwarderSpec `json:"clusterLogForwarder,omitempty"`
+}
+
+type OpenTelemetrySpec struct {
+	// +optional
+	Collector OpenTelemetryCollectorSpec `json:"collector,omitempty"`
+	// +optional
+	Instrumentation InstrumentationSpec `json:"instrumentation,omitempty"`
+}
+
+type OpenTelemetryCollectorSpec struct {
+	// +optional
+	Enabled bool `json:"enabled,omitempty"`
+}
+
+type InstrumentationSpec struct {
+	// +optional
+	Enabled bool `json:"enabled,omitempty"`
+}
+
+type UserWorkloadTracesSpec struct {
+	// +optional
+	OpenTelemetry OpenTelemetrySpec `json:"opentelemetry,omitempty"`
```

Comment:
to match the ADR https://docs.google.com/document/d/1Yha0IvwmTOzc3HMqG0gj9edRjlPIzIAwvotIFqCBJ9E/edit
```suggestion
	// +optional
	Collection OpenTelemetrySpec `json:"collection,omitempty"`
```